### PR TITLE
Update CONTRIBUTING.md with new way to run coverage check [skip ci]

### PR DESCRIPTION
**Pull request by zjmiller**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | no
| Major: Breaking Change?  | no
| Minor: New Feature?      | no
| Deprecations?            | no
| Spec Compliancy?         | no
| Tests Added/Pass?        | no
| Fixed Tickets            | none
| License                  | MIT
| Doc PR                   | yes
| Dependency Changes       | no

CONTRIBUTING.md referred to an old and no longer used way to run the coverage check. I just updated it to reflect the current Makefile.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ $ TEST_DEBUG=true make test
 To test the code coverage, use:
 
 ```sh
-$ make test-cov
```
Comment (hzoo) [inline CONTRIBUTING.md]:
```
test-ci-coverage:
	NODE_ENV=test BABEL_ENV=cov make bootstrap
	./scripts/test-cov.sh
	./node_modules/.bin/codecov -f coverage/coverage-final.json
```

This is for the CI so it runs make bootstrap which you wouldn't want to do each time you ran the coverage.

If we want to do this we should probably make a new command like `make test-cov` to only run `./scripts/test-cov.sh`

Comment (zjmiller) [inline CONTRIBUTING.md]: Ah that makes sense. I'll close this and just create an issue pointing out the inconsistency between CONTRIBUTING.md and the current Makefile.

Comment (existentialism) [inline CONTRIBUTING.md]: @zjmiller up for sending a PR that updates CONTRIBUTING.md to simply call `./scripts/test-cov.sh` directly?

Comment (zjmiller) [inline CONTRIBUTING.md]: Sure

**Pull request closed without merging**